### ISSUE TITLE
150ms delay during updateActiveEffect hook is somehow removing concentration

### ISSUE THREAD

**Issue by BenChand** [otigon/automated-jb2a-animations#444]
I'm currently running midi-qol (0.9.62) with DFred's convenient effects (2.6.3) and automated animations (3.2.91). When trying to use a spell that applies both a convenient effect from the spell as well as a convenient effect from concentration, automated animations seems to somehow cause the concentration to be removed. From some debugging locally, it seems to struggle to find the concentration effect after waiting 150ms here:

https://github.com/otigon/automated-jb2a-animations/blob/main/src/active-effects/ae5e.js#L21

If I remove this delay or increase it to, say, 1000ms, it correctly finds the concentration effect and doesn't cause concentration to be removed from the target. I'm wondering if the use of convenient effects and its replacement of the DAE concentration effect with its own effect is creating a sweet spot with no concentration effect that the automated animations library hits due to this 150ms delay (?)

**Comment by otigon** [comment]
The 150ms delay prevents errors from core Foundry due to trying to update an active effect at the same time it is being applied. I've had a few reports of this causing some issues with other aspects too, and I plan to see about removing any updates from my end, which should resolve this and other issues

**Comment by otigon** [comment]
Yeah looking at it now, I don't need to provide any updates to the Active Effect. I'll get that removed and added to the next release

**Comment by otigon** [comment]
Should be resolved in the currently released version of A-A
